### PR TITLE
Minor Fixes - CHS Issue 30

### LINKED ISSUE #274: Update link text to libcomcat on main Search page
The link text at the top of the Search page currently is:
Developer's Corner - bulk access to catalog, tools for obtaining specific products 

I suggest changing it to reflect the current "name" and "tagline" in github:
libcomcat - library of functions and wrapper scripts for accessing NEIC ComCat server data

### LINKED ISSUE #290: Fix arbitrary decimals in output
In more recent versions of PHP, the `json_encode` method serializes numbers differently. This has resulted in arbitrarily long and misleading precision on the values output in the various GeoJSON formats. [Read more here](https://stackoverflow.com/questions/42981409/php7-1-json-encode-float-issue/43056278)

As the post suggests, it seems this can be corrected by setting an INI variable.
```
serialize_precision = -1
```

Open to other fixes as well.

### LINKED ISSUE #291: Update KML namespace
reported via email 

> There is one small issue with the KML namespace that is used (eg  https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/1.0_week_age.kml) 
>
> Currently it is xmlns="http://earth.google.com/kml/2.2" which is not accepted by some newer parsers which insist on the official v2.2 namespace xmlns="http://www.opengis.net/kml/2.2"

### LINKED ISSUE #309: Update search page magnitude limits
minimum and maximum magnitude inputs are currently limited to the range [-1, 10].

- [ ] Change minimum and maximum magnitude inputs to allow values in the range [-10, 10]

https://github.com/usgs/earthquake-event-ws/blob/master/src/htdocs/inc/search-basic.inc.php#L10

### PR DESCRIPTION
**Pull request by ericklatzco**
Closes #309
Closes #291
Closes #290
Closes #274

### CODE REVIEW

**Comment by ericklatzco** [comment]
For #290, the serialization issue was fixed within the app rather than for the overall server (?). It would have required changes to httpd-php, realtime, and puppet, so the smaller/faster solution was taken.

**Comment by emartinez-usgs** [comment]
Can you please rebase this against the current upstream/master?

**Comment by ericklatzco** [comment]
> Can you please rebase this against the current upstream/master?

Should be done (I think?) 
Didn't have to actually do anything beside `rebase` and `git rebase,` so I don't actually know if it's good.

**Comment by emartinez-usgs** [comment]
I think the branch still says it is out of date. Try the following:
```
git  checkout master
git pull --ff-only upstream master
git push origin master
git checkout gitlab-issue-30
git rebase master
git push --force origin gitlab-issue-30
```

**Comment by ericklatzco** [comment]
Okay. Should hopefully be ready now.

File: src/htdocs/search.php [hunk 1]
```diff
@@ -55,8 +55,8 @@
           </li>
           <li>
             <a href="https://github.com/usgs/devcorner">
-              Developer\'s Corner - bulk access to catalog, tools for obtaining
-              specific products
+              Developer\'s Corner - Library of functions and wrapper scripts for
```
Comment (emartinez-usgs) [inline src/htdocs/search.php:58]: These apostrophes should be `&rsquo;`

File: src/htdocs/search.php [hunk 2]
```diff
@@ -55,8 +55,8 @@
           </li>
           <li>
             <a href="https://github.com/usgs/devcorner">
-              Developer\'s Corner - bulk access to catalog, tools for obtaining
-              specific products
+              Developer&rsquo;s Corner - Library of functions and wrapper scripts for
+              accessing and using tools for the NEIC\'s ComCat data
```
Comment (emartinez-usgs) [inline src/htdocs/search.php]: This one too: `&rsquo;`

File: src/htdocs/search.php (hunk 1 above)
Comment (ericklatzco) [reply src/htdocs/search.php]: Fixed in e28717d

**Pull request merged**